Sheepish-Squirrels/bangazon-api-sprint1
# Added GET, POST, PUT, and DELETE functionality to Training Programs end points

Fixes #9

Changes proposed in this pull request:

- created file to generate training program data using fakerjs and store in JSON
- added functionality to add training program data to sql table
- created routes for training program data
- created promises for GET, POST, PUT, DELETE training programs in model folder
- created controller for training program data to retrieve the data needed

Steps to test:

In terminal,
run nodemon app.js
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
Make sure you can GET all training programs, GET one training program, POST a training program, PUT (edit) a training program, and DELETE a training program
